libavformat/dashenc.c

```diff
@@ -1909,9 +1909,10 @@ static int dash_flush(AVFormatContext *s, int final, int stream)
 
         if (!os->bit_rate) {
             // calculate average bitrate of first segment
-            int64_t bitrate = (int64_t) range_length * 8 * AV_TIME_BASE / av_rescale_q(os->max_pts - os->start_pts,
-                                                                                       st->time_base,
-                                                                                       AV_TIME_BASE_Q);
+            int64_t rescale = av_rescale_q(os->max_pts - os->start_pts, st->time_base, AV_TIME_BASE_Q);
+            int64_t bitrate = 0;
+            if (rescale != 0)
```

Comment:
Change looks good.  I would wonder if it would be more correct to say > 0 instead of != as a negative number would probably be an error as well?